whatsupyoutube/main.py

```diff
@@ -29,36 +31,88 @@ def create_df(search_results) -> pd.DataFrame:
 def main() -> None:
     load_dotenv()
 
-    api_key = get_env_var("API_KEY")
 
-    channel_id = "UCVRobwJfa8nHMfJI3C8Rgdw"
-    page_token = ""
 
+    #api url components
+    api_key = get_env_var("API_Key")
+
+    channel_id = "UCX6OQ3DkcsbYNE6H8uQQuVA"
+    
+
+    #constructing the search api url
     base_url = "https://www.googleapis.com/youtube/v3"
     search_url = (
         f"search?key={api_key}"
         f"&channelId={channel_id}"
-        f"&part=snippet,id&order=date&maxResults=1000{page_token}"
+        f"&part=snippet,id&order=date&maxResults=50"
+        
     )
 
+
+
     request_url = f"{base_url}/{search_url}"
 
     search_response = requests.get(request_url).json()
-    search_snippet = search_response["items"]
+    
+    next_page_token = search_response['nextPageToken']
 
+    next_search_url = (
+        f"search?key={api_key}"
+        f"&channelId={channel_id}"
+        f"&part=snippet,id&order=date&maxResults=50"
+        f"&pageToken={next_page_token}"
+        
+    )
+
+    next_search_response = f"{base_url}/{next_search_url}"
+    next_page_search = requests.get(next_search_response)
+
+    #while loop to page all the way through
+    """while next_page_token is not None:
+        next_page_search = next_search_url"""
```

Comment:
Is there a reason you created a multi-level string instead of commenting it out?

Reply:
Figured using a string comment for a block of code is easier/makes more sense than commenting each line

Reply:
The Python style guide has users place a `#` before each line. Multi-level strings are typically used for docstrings. The closing quotes are placed on their own line.

https://peps.python.org/pep-0008/#block-comments